File: C/linearsearch.c
#include<stdio.h>
void display(int arr[],int size){
    printf("array is : ");
    for(int i=0;i<11;i++){
    printf("%d ",arr[i]);
    }
    printf("\n");
}
void linear(int arr[],int size,int element){
    for(int i=0;i<size-1;i++){
        if(element==arr[i]){
            printf("index is %d",i);
        }
    }

}
int main(){
    int arr[]={4,8 ,13,18,29,48,58,73,95,97,123};
    int size = sizeof(arr) / sizeof(int);
    display(arr,size);
    int element=58;
    linear(arr,size,element);
    return 0;
}

File: DSA/Arrays/LinearSearch.java
import  java.util.Arrays;

public class LinearSearch {
    static int FindElement(int[] arr,int element){
        for (int i=0;i<arr.length;i++) {
            if(arr[i]==element){
                if(i>0){
                    int temp=arr[i];
                    arr[i]=arr[i-1];
                    arr[i-1]=temp;
                    return i-1;
                }
                return i;
            } 
        }
        return -1;
    }
    public static void main(String[] args) {
        int[] arr = new int[] {23,43,56,76,45,93,65,43};
        int element =76;

        int result = FindElement(arr,element);
        System.out.println("Modify element is : "+Arrays.toString(arr));
        if (result==-1){
            System.out.println("element not found!");
        }else{
            System.out.println("Element index is : "+result);
        }
    }
}
